src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.cs

```diff
@@ -286,7 +286,7 @@ public static string Unescape(string str)
         /// <summary>
         /// Returns the regular expression pattern passed into the constructor
         /// </summary>
-        public override string ToString() => pattern;
+        public override string ToString() => pattern!;
```

Comment:
For cases where a helper is always called from the constructors, we've opted to make the fields always initialized (it's just not obvious to the compiler) as non-nullable.  Looks like this is one of those cases?

Reply:
Actually, no. There's also this guy: https://github.com/dotnet/corefx/blob/ffb94ad2c2d7c14fd9be82d365a28a60725b4f9f/src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.cs#L42-L45.